src/server/img_storage.rs

```diff
@@ -213,30 +213,4 @@ impl StoreImage {
             }
         }
     }
-
-    /// Download an image from a given [uri::Uri] and store it to Google Cloud Storage
-    pub async fn fetch(&self, uri: &uri::Uri) -> HandlerResult<Option<StoreResult>> {
```

Comment:
Was the removal of this function intentional?

Reply:
Yes. It's equivalent to the `store()` function.